### ISSUE TITLE
[Discussion] High Scores

### ISSUE THREAD

**Issue by jwilde35** [ermiyaeskandary/Slither.io-bot#301]
<h2>NOTE : ALL GAMES SHOULD HAVE THE SAME SETTINGS THROUGHOUT ~ ErmiyaEskandary</h2>

is it okay to make an issue about our high scores?? like boasting about our high scores and what settings we have the bot on?

if so, these are my settings/ highscores from last night, take note of the high score XD
![image](https://cloud.githubusercontent.com/assets/15678880/15941754/912465e8-2e4f-11e6-9f32-704c22c82de1.png)

**Comment by ermiyaeskandary** [comment]
Did you achieve that highscore with those settings throughout ?
@clemens-tolboom PR which records the time as well would be useful in this case.

**Comment by Fredyy90** [comment]
80k is my highest score so far.

![2016-06-08_2158](https://cloud.githubusercontent.com/assets/1572327/15964192/88c030f8-2f17-11e6-8df9-3326e382b134.png)

**Comment by ermiyaeskandary** [comment]
@Fredyy90 Did you use the same settings all through out ?

**Comment by Fredyy90** [comment]
Yes didn't change a thing, just had the window in the background running for ~72-96 hours

**Comment by jwilde35** [comment]
i left the bot on over the weekend at my school and these were the results
![image](https://cloud.githubusercontent.com/assets/15678880/15967562/3349c902-2ef6-11e6-993e-e287c8d52dea.png)

note: this was using the task execution mod

**Comment by clemens-tolboom** [comment]
@jwilde35 nice so those huge highscores for my #275 but I'm really puzzled in how to understand any scoring data.

Having an average is nice to have. Having a median too but what does it say? Using the data till now
- from 192 game average 9276 and median 5428 can still tell not enough about the bot performance. We do not know how bad the scores below the median are. I've seen lots of 12-40 points in some runs myself. We really need more data #286

@Fredyy90 ypur scoring has no median in it Guess you used an older version right?

**Comment by Seple** [comment]
Please test settings #290

**Comment by clemens-tolboom** [comment]
I just ran 4 concurrent (**develop**) bots for 2:12 hours (and recorded it into a 160GB video make system low on memory)

![image](https://cloud.githubusercontent.com/assets/371014/15984886/e6482104-2fda-11e6-86c8-5bb9b02fcf19.png)

![image](https://cloud.githubusercontent.com/assets/371014/15984891/fd805800-2fda-11e6-9ab6-84568ab532f6.png)

![image](https://cloud.githubusercontent.com/assets/371014/15984892/0229143c-2fdb-11e6-8e00-5dce3ef509f3.png)

![image](https://cloud.githubusercontent.com/assets/371014/15984893/05325346-2fdb-11e6-89aa-e3e476664245.png)

| # | a | m | max |
| --- | --- | --- | --- |
| 21 | 3914 | 2378 | 14510 |
| 10 | 6989 | 4867 | 26334 |
| 13 | 6449 | 1467 | 26959 |
| 17 | 5660 | 3271 | 18321 |

**Comment by grovtor** [comment]
Can scores be saved to the log or a desktop text file?

@jwilde35 how does one install that?  Explain it to me as if I were 14

**Comment by ermiyaeskandary** [comment]
@grovtor If you mean the task execution feature, install [this](https://github.com/ErmiyaEskandary/Slither.io-bot/raw/53460469ef01e554b90cbae7525944886807f508/bot.user.js)

**Comment by clemens-tolboom** [comment]
@grovtor I use the HTML inspector then select the scores and paste it into a text file.

**Comment by tjorim** [comment]
For testing, and easy score saving, I would suggest to use https://github.com/K00sKlust/slither.io-bot-manager .

**Comment by clemens-tolboom** [comment]
I talked to a statisticus and he suggests it's not about the absolute score but the score made relative to the others. It is of no use to register _1000_ as a bad score when you end up as # 1 where # 2 has _300_ points.

That why I plead for more score info in #286

**Comment by ChadSki** [comment]
I'm not sure I agree that being #1 on the server with just 1000 points is very impressive. Ideally the bot simply doesn't die that small, and continues to grow even if other snakes are small.

Since we only consider comparing scores valid if you're running both bots at the same time, they should be facing roughly equal server conditions anyways.

In theory collecting extra information won't harm anything, but I'm concerned that people will be using the wrong information to make judgements.

But, I guess it would be interesting to see. ¯_(ツ)_/¯

**Comment by phrokton** [comment]
The _SnakeParamsBySize_ of the Task Execution branch seems like it would be best for unattended use especially as snake grows to the point where turning requires an absurd radius.

edit:  oh,  https://github.com/ErmiyaEskandary/Slither.io-bot/issues/290

I have not yet convinced myself I can edit that section to create more size brackets because it is not clear if those brackets connect to some other part of the code.

**Comment by clemens-tolboom** [comment]
@phrokton huh ?!??

**Comment by jwilde35** [comment]
@phrokton you can... i have but did not save it... works fine
